src/ccstruct/tabletransfer.h

```diff
@@ -0,0 +1,66 @@
+/******************************************************************************
```

Comment:
completely suspicious file. We have more global vars now. I prefer to put them inside some owner data struct (like TessAPI or Tesseract). Is it possible?